posts/2024-07-23_myg-10/README.md

```diff
@@ -0,0 +1,50 @@
+---
+publication_date: 2024-07-22T00:00:00Z
+slug: monthly-dev-10
+tags: [gnoland, ecosystem, updates, gnovm, tm2, test4, gnostudio, connect]
+authors: [Kouteki]
+---
+
+# The More You Gno 10: Test4 is Live!
+
+This edition focuses on Test4, a major milestone towards our mainnet. Test4 is the first true multinode testnet featuring DAOs and on-chain governance, offering a preview of what’s to come. Much excitement, such wow.
+
+We document everything in our weekly engineering updates and video recordings.
+
+# Gno Core Updates
+
+## Test4 is Live
+
+With 7 validator nodes running and 3 more about to be added via on-chain governance, Test4 is the first multinode testnet that has the complexity and the feature set we want for the gno.land mainnet. Find out more about it in [this article](https://www.gno.land/r/gnoland/blog:p/test4-live).
+
+## Changelog
+
+- Gnoweb live
+- RPC live
+- TX indexer available
+- Test4 faucet added to the [Faucet Hub]](https://faucet.gno.land/)
+- Merged in [Gno type check](https://github.com/gnolang/gno/pull/1426) support, resolving the long-standing issue with Gnolang's type checking on a VM level, making it more stable for development.
+- Added support for [transpiling gno standard libraries](https://github.com/gnolang/gno/pull/1695), as part of a bigger effort to stabilize the GnoVM with native binding support (which was added a while back). The Gno transpiler now uses Gno's standard libraries instead of native Go's. This also eliminates the need for things like `stdshim` and an std whitelist.
```

Comment:
```suggestion
- Added support for [transpiling gno standard libraries](https://github.com/gnolang/gno/pull/1695), as part of a bigger effort to stabilize the GnoVM with native binding support (which was added a while back). The Gno transpiler now uses Gno's standard libraries instead of Go's. This also eliminates the need for things like `stdshim` and an std whitelist.
```
We refer to them as native internally but this might not make much sense to others reading this.